ocs_ci/deployment/deployment.py

```diff
@@ -370,12 +386,54 @@ def subscribe_ocs(self):
         if subscription_plan_approval == "Manual":
             wait_for_install_plan_and_approve(self.namespace)
 
+    def get_zone_label(self):
+        nodes = ocp.OCP(kind="node").get().get("items", [])
+
+        # Check which zone label is present
+        return (
+            constants.ZONE_LABEL
+            if nodes[0]["metadata"]["labels"].get(constants.ZONE_LABEL, False)
+            else constants.ZONE_LABEL_NEW
+        )
+
+    def get_arbiter_location(self):
+        """
+        Get arbiter mon location for storage cluster
+        """
```

Comment:
Can't you just ask for a node in arbiter zone?

Reply:
That can be done, In that case everything needs to be done by user/admin